src/components/Dashboard/Role/Users.js

```diff
@@ -0,0 +1,105 @@
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { bindActionCreators } from 'redux';
+import { Alert, Table } from 'react-bootstrap';
+import {
+  DashboardLink as DashLink,
+  DashboardLayout as Layout
+} from 'components/Dashboard/';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+import { actions as rolesActions } from 'reducers/roles';
+import { getRoleUsers } from 'selectors/roles';
+
+export class RoleUsers extends Component {
+  static propTypes = {
+    actions: PropTypes.object.isRequired,
+    opt: PropTypes.object.isRequired,
+    roleId: PropTypes.number,
+    name: PropTypes.string,
+    roleUsers: PropTypes.array
+  };
+  constructor(props) {
+    super(props);
+    this.updateRoleId = this.updateRoleId.bind(this);
+  }
+  updateRoleId(roleId) {
+    const { actions } = this.props;
+
+    actions.requestRoleUsers({ roleId });
+  }
+  componentDidMount() {
+    const { actions, roleId } = this.props;
+
+    actions.requestRoles();
+    if (roleId !== null) {
+      actions.requestRoleUsers({ roleId });
+    }
+  }
+
+  render() {
+    const { name, opt, roleId, roleUsers } = this.props;
+
+    return (
+      <Layout
+        pageTitle="Role Users - Dashboard"
+        header={`Roles > ${name}`}
+        options={opt}
+      >
+        <FontAwesomeIcon icon="chevron-left" /> &nbsp;
+        <DashLink to="#roles" name="Roles">
+          Back
+        </DashLink>
+        {!roleUsers && <Alert>No Users Assigned to this Role!</Alert>}
```

Comment:
You should use a ternary here or else odd behavior can occur where both are displayed for a brief moment.